python/lsst/cp/pipe/ptc.py

```diff
@@ -255,9 +254,9 @@ def __setattr__(self, attribute, value):
             self.__dict__[attribute] = value
 
     def getVisitsUsed(self, ampName):
```

Comment:
We're not really using visits anywhere, right?  Can we drop it everywhere in favor of 'exposure', or should that be a separate ticket?  We'll want to do it at some point.